src/test/java/com/quizapp/enterprise/EnterpriseApplicationTests.java

```diff
@@ -76,11 +76,11 @@ void verifyCorrectAnswer() {
         ArrayList<Guess> gec = new GameService().checkGuess(userGuess, correctAnswer);
         boolean correct = true;
         for(Guess guess : gec) {
-
           if(!guess.IsCorrectLetter) {
-                fail("User Guess of "
-                        + guess.Letter
-                        + " was not correct");
+              StringBuilder failedLetterGuessString = new StringBuilder("User Guess of ")
+                      .append(guess.Letter)
+                      .append(" was not correct");
+                fail(failedLetterGuessString.toString());
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/